### DISCUSSION TITLE
Creating and accessing encrypted SQLite database

### DISCUSSION THREAD

**Discussion by YogiYang** [MagicFoundation/Alcinoe#208] [Q&A]
Hello,

Is it possible to create and access SQLite database that is encrypted from Alcinoe's SQLite component?

**Comment by Zeus64** [comment]
Not yet :( but I don't think it's will be complicated to add this support (I doubt it's will take more than very few lines of codes)

**Reply by YogiYang** [reply]
Can you help me achieve this.

If I am compiling my project for Linux what will I have to distribute with my application to access SQLite?
